_inc/lib/core-api/wpcom-endpoints/publicize-connections.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+/**
+ * Publicize: List Connections
+ *
+ * [
+ *   { # Connnection Object. See schema for more detail.
+ *     id:           (string)  Connection unique_id
```

Comment:
Question: is there a good reason why `unique_id` is a string?

Reply:
Just that I don't trust Jetpack and WordPress.com both to always return an integer. It's supposed to be an opaque value anyway, so using a string gives us some future flexibility.

Also, if we ever pass 4294967295 connections, an integer would break some clients :) (I'm sure there are other fields where this is much more likely to happen :)).